datadog/resource_datadog_security_monitoring_rule.go

```diff
@@ -432,6 +477,7 @@ func updateResourceDataFromResponse(d *schema.ResourceData, ruleResponse datadog
 	d.Set("case", ruleCases)
 	d.Set("message", ruleResponse.GetMessage())
 	d.Set("name", ruleResponse.GetName())
+	d.Set("has_extended_title", ruleResponse.GetHasExtendedTitle())
```

Comment:
Is there a test which checks this change?

Reply:
well it should be in the previous PR actually 👀

Reply:
I'm creating a separate PR to fix this, let's remove from this one
```suggestion
```